config.py

```diff
@@ -446,7 +446,7 @@
 # >>> conditions = ['left', 'right']
 
 # conditions = ['left', 'right']
-conditions = None
+conditions = [None]
```

Comment:
@jasmainak this is where it breaks now. perhaps you can figure out the conditions for decoding? There is no `events.tsv` file, so I guess one would have to read in the file and check.

But I need to call it a day for today

Reply:
see my related comment here: https://github.com/mne-tools/mne-study-template/pull/40#issuecomment-518913104. MNE needs to throw more informative errors.